src/Symfony/Component/Yaml/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.4
+---
+
+ * Add support for getting all the enum cases with `!php/enum Foo` and `!php/enum Foo::cases()`
```

Comment:
IMO we should settle on one of the two forms. Having two ways to achieve the same is rather confusing.

Reply:
Indeed. If  `!php/enum Foo::cases()`, this will lead to confusion as it looks like any static method would be accepted.

Reply:
It's removed. Thank you.